README.md

```diff
@@ -889,7 +891,14 @@ For flags defined in `appsettings.json`, that is done by using the `TelemetryEna
 }
 ```
 
-The appsettings snippet above defines a flag named `MyFlag` that is enabled for telemetry.
+The appsettings snippet above defines a flag named `MyFlag` that is enabled for telemetry. This is indicated by the `Telemetry` object which sets `Enabled` to true. The value of the `Enabled` property must be `true` to publish telemetry for the flag.
+
+The `Telemetry` section of a feature flag has the following properties:
+
+| Property | Description |
+| ---------------- | ---------------- |
+| `Enabled` | Specifies whether telemetry should be published for the feature flag. |
+| `Metadata` | A collection of key-value pairs, modeled as a dictionary, that can be used to add custom metadata on evaluation events that are generated by the flag. |

```

Comment:
I feel confusing about the word "generated". My understanding is that the metadata dictionary usually works like custom tag/label to provide some extra information about the feature flag.

I would suggest something like " ..., that can be used to attach custom metadata about the feature flag to evaluation events."

Reply:
I like the suggestion.

Reply:
Updated.